docs/config/cleanup.md

```diff
@@ -20,3 +20,8 @@ If true the scm checked out files will be wiped.
 If true wipes the build workspace.
 
 **Default:** false
+
+### CLEAN_MINION_DATA
+If true wipes the minion yamls directory.
+
```

Comment:
Since it is always confusing when what happens, can we add here to the readme a qualifier like "at the end of the build"? Also same for CLEAN_BUILD_WORKSPACE. And for CLEAN_SCM_WORKSPACE a "at the beginning of the build"?